CONIC_SUBROUTINES.s

```diff
@@ -1825,7 +1825,7 @@ KEPC2		EQUALS	36D
 # R2VEC		ERASE 	+5
 # TDESIRED	ERASE	+1
 # GEOMSGN	ERASE	+0
-# GUESSW			# 0 IF COGA GUESS AVIABLE, 1 IF NOT
+# GUESSW			# 0 IF COGA GUESS AVAILABLE, 1 IF NOT
 # COGA		ERASE	+1	# INPUT ONLY IF GUESS IS ZERO.
```

Comment:
Typo that nobody has spotted yet ;)

`GUESS` needs changing to `GUESSW` [Source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1306.jpg)